src/odemis/gui/util/__init__.py

```diff
@@ -292,7 +292,9 @@ def enable_tab_on_stage_position(button, stage, pos, target, aligner=None, toolt
     :param aligner: (Actuator) the align component
     :param tooltip: (str or None) Tooltip message to show when disabled
     """
-    within_target = getCurrentPositionLabel(pos, stage, aligner) in target
+    microscope = model.getMicroscope()
+    microscope_manager = MicroscopePostureManager(microscope)
+    within_target = microscope_manager.getCurrentPostureLabel(pos) in target
```

Comment:
No, don't create a new posture manager everytime. Instead, use the one from the GUI, by taking it as an argument. Note also that the stage, pos and aligner arguments are not used anymore.
=> change the signature to `button, posture_manager, target, tooltip=None`.
Don't forget to update every